djangoappengine/static/js/guestbook/widget/guestbookWidget.js

```diff
@@ -1,22 +1,107 @@
 define([
-    'dojo/_base/declare',
-    'dojo/_base/fx',
-    'dojo/_base/lang',
-    'dojo/dom-style',
-    'dojo/mouse',
-    'dojo/on',
-    'dijit/_WidgetBase',
-    'dijit/_TemplatedMixin',
-    'dojo/text!./templates/myModule.html'
-], function(declare, baseFx, lang, domStyle, mouse, on, _WidgetBase, _TemplatedMixin, template){
-
-    return declare([_WidgetBase, _TemplatedMixin], {
-        templateString: template,
+	'dojo/_base/declare',
+	'dojo/request',
+	"dojo/dom",
+	'dojo/request/xhr',
+	'dojo/_base/fx',
+	'dojo/_base/lang',
+	'dojo/dom-style',
+	'dojo/mouse',
+	'dojo/on',
+	'dijit/_WidgetBase',
+	'dijit/_TemplatedMixin',
+	'dojo/text!./templates/myModule.html'
```

Comment:
Rearrang import library.
Follow this: http://redmine.asv.vn/redmine/boards/23/topics/31-source-code-review-checklist